recipes/apptainer/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set version = "1.1.0" %}
+
+package:
+  name: apptainer
+  version: {{ version }}
+
+source:
+  - folder: src/github.com/apptainer/apptainer
+    url: https://github.com/apptainer/apptainer/releases/download/v{{ version }}/apptainer-{{ version }}.tar.gz
+    sha256: 1772f25c8770f976abb82c29053b77183930426a83eb3e43b9f8068c3e88ac30
+    patches:
+      #- patches/0003-Use-external-CNI.patch
+      #- patches/0004-Find-unsquashfs-from-PATH.diff
+      #- patches/set-version-from-env.patch
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  # The binary is made runtime relocatable using the included patch
+  detect_binary_files_with_prefix: false
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - go >=1.16
+    - sysroot_linux-64 2.17  # [linux64]
+    - make
+    - pkg-config
+    - go-dep
+    - sed
+  host:
+    - libuuid  # [linux]
+    - libarchive
+    - libseccomp  # [linux]
+    - openssl
+  run:
+    # CNI version from singularity/Gopkg.lock
+    - cni-plugins >=0.7.4  # [linux]
+    - squashfs-tools  # [linux]
+  run_constrained:
+    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx]
```

Comment:
This will only ever be linux-only so no need for this:
```suggestion
```

Reply:
True, I wonder why this recipe ever had osx stuff in it since singularity never ran on osx.